index.html

```diff
@@ -0,0 +1,30 @@
+<html><head>
+  <title>Grocery List</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+<div id="main">
+  <img src="grocery-bags.png">
+  <div id="app">
+    <h1>Grocery List</h1>
+    <div>
+      <input value="">
+      <input value="">
+      <button>Add Grocery</button>
+    </div>
+    <div class="groceries">
+      <div>
+        <span> frozen pizza </span>
+        <span> 5 </span>
+      </div>
+      <div>
+        <span> noosa yogurt </span>
+        <span> 10 </span>
+      </div>
+    </div>
+  </div>
+</div>
```

Comment:
the above appears to be an unordered list. You should refactor this to use `<ul>`'s and `<li>`'s